tests/components/lifx/test_button.py

```diff
@@ -21,6 +25,13 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.fixture(autouse=True)
+def mock_lifx_coordinator_sleep():
+    """Mock out lifx coordinator sleeps."""
+    with patch("homeassistant.components.lifx.coordinator.asyncio.sleep"):
```

Comment:
Please patch the time slept instead of patching asyncio sleep. This is usually easier to do when storing the sleep time in a constant.